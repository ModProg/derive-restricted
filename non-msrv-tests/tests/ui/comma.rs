#[derive(derive_where::DeriveWhere)]
#[derive_where(Clone,; T)]
struct Test1<T>(T);

#[derive(derive_where::DeriveWhere)]
#[derive_where(Clone,,)]
struct Test2<T>(T);

#[derive(derive_where::DeriveWhere)]
#[derive_where(Clone; T;)]
struct Test3<T>(T);

#[derive(derive_where::DeriveWhere)]
#[derive_where(Clone; T,,)]
struct Test4<T>(T);

fn main() {}
