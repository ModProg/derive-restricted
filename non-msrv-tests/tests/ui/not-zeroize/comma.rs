#[derive(derive_where::DeriveWhere)]
#[derive_where(Clone,,)]
struct Test<T>(T);

fn main() {}
