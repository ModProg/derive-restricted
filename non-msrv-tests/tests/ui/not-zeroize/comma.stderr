error: unsupported trait syntax, expected one of expected one of Clone, Copy, Debug, Default, Eq, Hash, Ord, PartialEq, PartialOrd
 --> tests/ui/not-zeroize/comma.rs:2:22
  |
2 | #[derive_where(Clone,,)]
  |                      ^
